_episodes/09-pull-requests.md

```diff
@@ -192,99 +291,224 @@ Largest City: Paris
 {: .output}
 
 Next we will add and commit these changes.
-Then we will push them to our forked version of the repo.
```

Comment:
There are quite a few places (some of them legacy) where "forked version" is used.

I think I prefer just using the term `fork` instead to get them familiar with that nomenclature. Maybe mixing this term in or providing extra description in parenthesis once or twice for clarity is a nice middle ground.